modules/ROOT/pages/_partials/task-flex-add-tls-context.adoc

```diff
@@ -59,9 +59,9 @@ By default, *H2 - HTTP/1.1* are the selected ALPN protocols. Change this value t
 * *Inbound Settings* +
 Optionally, select *Validate Client certificate* to support mTLS for inbound traffic.
 * *Outbound Settings* +
-Optionally, select *Skip server certificate validation* to skip the validation of the outbound server.
+Optionally, select *Skip server certificate validation* to skip the validation of the outbound server and not support mTLS for outbound traffic.
```

Comment:
mTLS does not depend on this check for outbound, but rather on the use of a keystore. If the user provides a keystore and the TLS context is used for outbound, then Flex will use the certificate for the communication (i.e. it will initiate a client side mTLS connection).